build/exe/doliwamp/index.php.install

```diff
@@ -527,7 +527,7 @@ a:hover {
 </head>
 
 <body>
-<table width="100%"><tr><td align="left">
+<table width="100%"><tr><td style="text-align:left;">
```

Comment:
correct fix is to use
<td class="left">

The difference with style= is that with style=, it will always be left align. When you use class="left", it a css that depends on the user language. For right to left language, it will be automatically right instead of left.

So change must be one toward
class="left"